# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/platforms/agnostic/AOMDecoder.cpp
# Commit: fd9405ffb398
# Full Hash: fd9405ffb3980cba0058cb908c08707a50d977cb
# Author: Zaggy1024 <Zaggy1024@gmail.com>
# Date: 2022-04-12 21:36:55
# Description:
#   Bug 1764282 - Fix reading of AV1 sequence headers with reduced_still_picture set. r=alwu
#   
#   Certain YouTube videos end with a single AV1 sequence header with reduced_still_picture set. ReadSequenceHeaderInfo was returning an empty list of operating points, causing an assertion failure.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D143488
# ==============================================================================

diff -r 5df04fd063fc -r fd9405ffb398 dom/media/platforms/agnostic/AOMDecoder.cpp
--- a/dom/media/platforms/agnostic/AOMDecoder.cpp	Tue Apr 12 09:49:21 2022 -0700
+++ b/dom/media/platforms/agnostic/AOMDecoder.cpp	Tue Apr 12 17:04:17 2022 +0000
@@ -526,6 +526,7 @@
   br.ReadBit();  // enable_intra_edge_filter
 
   if (reducedStillPicture) {
+    aDestInfo = tempInfo;
     return true;
   }
 
